src/components/Option/index.js

```diff
@@ -51,17 +55,48 @@ class OptionItem extends Component {
     }
 
     handleClick(event) {
+        const { disabled, privateOnClick, label, name, icon, value, showCheckbox } = this.props;
+        if (showCheckbox) {
+            event.preventDefault();
+            event.stopPropagation();
```

Comment:
same ^^ we need to avoid stopPropagation

Reply:
Done